src/ffi/client.rs

```diff
@@ -103,6 +104,22 @@ ffi_fn! {
     } ?= std::ptr::null_mut()
 }
 
+ffi_fn! {
+    /// temp
+    fn hyper_clientconn_options_set_preserve_header_case(opts: *mut hyper_clientconn_options) {
```

Comment:
To follow the example of the other options that are boolean, perhaps we should have the user pass a `c_int` here as well?

Reply:
On it!

Reply:
Quick question about FFI in general:

When I was playing around with the code on the `curl` side of things I noticed that `hyper_clientconn_options_set_preserve_header_case` would get optimized out if I didn't add a `println!` to the function  (ie: I would call the function from curl, but to boolean value wouldn't change) . Is there something like `#[no_mangle]`  that I should add here?